Answers.md

```diff
@@ -1,11 +1,15 @@
 # Answers
 
 1.  What is React JS and what problems does it try and solve?
+    React is a javascript library used for creating rich user interfaces. Solves the problem

```

Comment:
It also solves the issue with having to reload the whole page and only  reloading the section you need.

Reply:
Everything looks good not much to say here since this is a review for you basically. Maybe next week there will be some stuff that you struggled with the first time and you can learn more the second time around.